script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_3.txt

```diff
@@ -206,9 +206,9 @@ So how could she wait for me in front of the school and smile like that?
 I killed her.
 I Killed Her.
 I KILLED HER.
-why, why, why, why, why, why, why, why, why?
-why, why, why, why, why, why, why, why, why, why, why?
-why, why, why, why, why, why, why, why, why, why, why――――――?
+Why, Why, Why, Why, Why, 
+Why, Why, Why, Why, Why, 
+Why, Why, Why, Why, Why――――――?!
```

Comment:
So we're putting capital W each time, even with the commas?

Reply:
I think it works better with capital yeah